code/lightna/lightna-engine/js/lib/Blocks.js

```diff
@@ -1,5 +1,5 @@
 import { $ } from 'lightna/lightna-engine/lib/utils/dom';
-import request from 'lightna/lightna-engine/lib/HttpClient';
+import { HttpClient } from 'lightna/lightna-engine/lib/HttpClient';
```

Comment:
just an idea/question, maybe make sense to call it shorter Http?

Reply:
maybe Request than?

Reply:
Sure, why not